### DISCUSSION TITLE
Anyone can tell me what is the Badging?

### DISCUSSION THREAD

**Discussion by Jaehwa-Noh** [android/nowinandroid#1357] [Q&A] [answered]
I found this [Badging.kt](https://github.com/android/nowinandroid/blob/main/build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/Badging.kt) file.
I can't figure out it does what.
Is there anyone can tell me what is it?

**Comment by SimonMarquis** [comment, marked as answer]
Here is the rationale: https://github.com/android/nowinandroid/pull/1001

**Reply by Jaehwa-Noh** [reply]
Thank you for answering.
I'll check it.

**Comment by alexvanyo** [comment]
https://android-developers.googleblog.com/2023/12/increase-your-apps-availability-across-device-types.html goes through a more detailed view of the overarching issue for apps.

**Reply by Jaehwa-Noh** [reply]
Thank you. I'll check it.
